.pre-commit-config.yaml

```diff
@@ -1,27 +1,37 @@
 repos:
-- repo: https://github.com/pycqa/isort
-  rev: 5.10.1
-  hooks:
-    - id: isort
-      name: isort (python)
-- repo: https://github.com/sourcery-ai/sourcery
-  rev: v0.13.0
-  hooks:
-  - id: sourcery
-    args: [--diff=git diff HEAD, --in-place, --no-summary]
-- repo: https://github.com/psf/black
-  rev: 22.10.0
-  hooks:
-  - id: black
-    language_version: python3.10
-- repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v3.4.0
-  hooks:
-    - id: mixed-line-ending
-      args:
-        - "--fix=lf"
-    - id: pretty-format-json
-      args:
-        - "--autofix"
-        - "--indent=4"
-        - "--no-sort-keys"
\ No newline at end of file
+  # import formatter with black configurations
+  - repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        name: isort (python)
+        args: ["--profile", "black", "--filter-files"]
+
+  # Code formatter for both python files and jupyter notebooks
```

Comment:
Really appreciated the commitment to documentation/readability with comments in this file!